demos/dermatology.html

```diff
@@ -87,7 +90,6 @@
                             <option value="src/amchart/Bar">AmChart Bar</option>
                             <option value="src/amchart/Bubble">AmChart Bubble</option>
                             <option value="src/amchart/Candle">AmChart Candle</option>
-                            <option value="src/amchart/FloatingColumn">AmChart Floating Column</option>

```

Comment:
is this a double ...? this shouldn't be here (prolly a diff between your copy and master) can you fix this since we need the other edits?